.github/workflows/build-ci.yml

```diff
@@ -23,6 +23,10 @@ jobs:
                     - '8.1'
                     - '8.2'
                     - '8.3'
+                include:
+                    - php: '8.1'
+                      mongodb: '5.0'
+                      mode: 'low-deps'
```

Comment:
@alcaeus this modifies an existing matrix combination instead of creating an extra one. I think it's perfect.